Formula/openshift-cli.rb

```diff
@@ -1,14 +1,20 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://github.com/openshift/oc.git",
-      tag:      "openshift-clients-4.6.0-202006250705.p0",
-      revision: "51011e4849252c723b520643d27d3fa164d28c61",
-      shallow:  false
-  version "4.6.0"
   license "Apache-2.0"
   head "https://github.com/openshift/oc.git"
 
+  stable do
+    url "https://github.com/openshift/oc.git",
+        tag:      "openshift-clients-4.6.0-202006250705.p0",
+        revision: "51011e4849252c723b520643d27d3fa164d28c61",
+        shallow:  false
+    version "4.6.0"
+
+    # Add Makefile target to build arm64 binary
+    patch :DATA
```

Comment:
Lets upstream this patch, please.

Reply:
I at least opened PR https://github.com/openshift/oc/pull/889, but it will probably depend on whether upstream wants to support that build.

Difference between v4.6.0 and current master (about 1 year since last tag) means still need to use local patch.